client/src/components/ItemDetail/ItemDetail.tsx

```diff
@@ -63,13 +83,31 @@ function ItemDetail({
     }
     // Everytime we change to a 'new' item, refresh removal state flag
     setIsRemoving(false);
+    // Refresh whether we collapse media (image) to show more of text
+    setMediaCollapsed(false);
+    // Determine whether we should render 'show more' expand button
+    const contentDiv = document.getElementById("item-detail-description");
+    if (contentDiv) setIsOverflow(checkContentOverflow(contentDiv));
+    else setIsOverflow(false); // Cannot find content tag, no content to render
```

Comment:
nit: Extra comments not needed here
```suggestion
    if (contentDiv) setIsOverflow(checkContentOverflow(contentDiv));
    else setIsOverflow(false); 
```

Reply:
removed.

Reply:
did you push you changes> I dont see them

Reply:
oops, was committing changes suggested here into another branch, since resolved. Should be able to see the changes now!